search/search-client-connect/src/main/java/io/camunda/search/connect/es/ElasticsearchConnector.java

```diff
@@ -86,6 +86,10 @@ public ElasticsearchAsyncClient createAsyncClient() {
     return new ElasticsearchAsyncClient(transport);
   }
 
+  public ObjectMapper objectMapper() {
+    return objectMapper;
```

Comment:
👍🏼 So we reuse now the ObjectMapper created in the search module, everywhere right?

Reply:
Yes, it's then pased everywhere from here